forward-message-sendgrid/README.md

```diff
@@ -1,3 +1,10 @@
+
+### WARNING !!! 
+### This Repo is now out of date. 
+
+With the release of Node v18, the Node.js ecosystem are migrating over from the old CommonJS (CJS) standard to the newer, ES Modules (ESM) standard. You can read about the differences in far more detail in this [Blog Post.](https://redfin.engineering/node-modules-at-war-why-commonjs-and-es-modules-cant-get-along-9617135eeca1). The following snippets may causes errors. 
```

Comment:
grammar: 
- "ecosystem are migrating" to "ecosystem is migrating"
- "Blog Post." to "blog post."
- "may causes errors" to "may cause error"

Thanks!